pkg/application/app.go

```diff
@@ -635,3 +639,48 @@ func (app *Avalanche) CheckCertInSSHDir(certName string) (bool, error) {
 func (app *Avalanche) GetAnsibleInventoryPath(clusterName string) string {
 	return filepath.Join(app.GetNodesDir(), constants.AnsibleInventoryDir, clusterName)
 }
+
+func (app *Avalanche) GetAnsibleStatusDir() string {
+	return filepath.Join(app.GetAnsibleDir(), constants.AnsibleStatusDir)
+}
+
+func (app *Avalanche) GetBootstrappedJSONFile() string {
+	return filepath.Join(app.GetAnsibleStatusDir(), constants.IsBootstrappedJSONFile)
+}
+
+func (app *Avalanche) GetNodeIDJSONFile() string {
+	return filepath.Join(app.GetAnsibleStatusDir(), constants.NodeIDJSONFile)
+}
+
+func (app *Avalanche) GetSubnetSyncJSONFile() string {
+	return filepath.Join(app.GetAnsibleStatusDir(), constants.SubnetSyncJSONFile)
+}
+
+func (app *Avalanche) SetupAnsibleEnv() error {
+	err := os.RemoveAll(app.GetAnsibleDir())
+	if err != nil {
+		return err
+	}
+	err = app.CreateAnsibleDir()
+	if err != nil {
+		return err
+	}
+	return app.CreateAnsiblePlaybookDir()
+}
+
+// CreateFile creates file named fileName in .avalanche-cli ansible status directory
+func (app *Avalanche) CreateFile(filePath string) error {
```

Comment:
probably name it CreateAnsibleStatusFile and only pass the filename not the full file path
or have one create per ansible status file kind